src/main/java/com/github/danielflower/mavenplugins/release/BaseMojo.java

```diff
@@ -197,4 +206,22 @@ static void printBigErrorMessageAndThrow(Log log, String terseMessage, List<Stri
         log.error("");
         throw new MojoExecutionException(terseMessage);
     }
+
+    protected static String getRemoteUrlOrNullIfNoneSet(Scm originalScm, Scm actualScm) throws ValidationException {
```

Comment:
This was used by both `ReleaseMojo` and `NextMojo`, so I moved it to the base class of them both. It's not otherwise involved in this PR.